lib/utilrb/pkgconfig.rb

```diff
@@ -74,13 +74,17 @@ def self.get(name, version_spec = nil, preset_variables = Hash.new, minimal: fal
 
             # Now try to find a matching spec
             if match = find_matching_version(candidates, version_spec)
-                match
+                if version_spec.eql? match
+                    return nil
+                else
+                    match
```

Comment:
This `match` - which was there before - looks like a leftover from old copy/paste code. It's useless.